api/all.yaml

```diff
@@ -35,6 +35,9 @@ tags:
   - name: Permissions
     description: |
       In Unity Catalog, data is secure by default. Access can be granted.
+  - name: CoordinatedCommits
+    description: |
+      "Coordinated commits" is a Delta tables feature that allows Delta tables to designate a coordinator to coordinate commits. This is Unity Catalog's API to allow Unity Catalog be the commit coordinator for managed tables.
```

Comment:
can you link to the delta protocol doc? 
https://github.com/delta-io/delta/blob/master/PROTOCOL.md